### PR TITLE
Add staticpod

### PR DESCRIPTION
**Pull request by luomengY**
#### Which issue(s) this PR fixes:
Fixes #https://github.com/kubeedge/kubeedge/issues/4778

#### Special notes for your reviewer:
@Shelley-BaoYue

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -232,6 +233,18 @@ type Dependencies struct {
 	useLegacyCadvisorStats bool
 }
 
+// makePodSourceConfig creates a config.PodConfig from the given
+// KubeletConfiguration or returns an error.
+func makePodSourceConfig(cfg *config.PodConfig, kubeCfg *kubeletconfiginternal.KubeletConfiguration, kubeDeps *Dependencies, nodeName types.NodeName, nodeHasSynced func() bool) error {
+	const kind = "KubeletConfiguration"
+	kubeCfg.StaticPodPath = kubeadmapiv1.DefaultManifestsDir
```
Comment (Shelley-BaoYue) [inline pkg/kubelet/kubelet.go]: suggest use kubeCfg.StaticPodPath directly.  The value can be setted in kubeedge edgecore.yaml

Comment (luomengY) [reply pkg/kubelet/kubelet.go]: Okay, I have made the changes.

File: cmd/kubeadm/app/apis/kubeadm/v1beta3/defaults.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ const (
 	// (previously this defaulted to k8s.gcr.io)
 	DefaultImageRepository = "registry.k8s.io"
 	// DefaultManifestsDir defines default manifests directory
-	DefaultManifestsDir = "/etc/kubernetes/manifests"
+	DefaultManifestsDir = "/etc/kubeedge/manifests"
```
Comment (Shelley-BaoYue) [inline cmd/kubeadm/app/apis/kubeadm/v1beta3/defaults.go]: As we set static path in edgecore.yaml. This modification can be removed.

File: pkg/kubelet/kubelet.go [hunk 3]
```diff
@@ -34,7 +35,6 @@ import (
 	cadvisorapi "github.com/google/cadvisor/info/v1"
 	libcontaineruserns "github.com/opencontainers/runc/libcontainer/userns"
 	"github.com/opencontainers/selinux/go-selinux"
-	"k8s.io/mount-utils"
```
Comment (Shelley-BaoYue) [inline pkg/kubelet/kubelet.go]: Keep the original order of import 
image

**Pull request closed without merging**